code/modules/reagents/chemistry/items.dm

```diff
@@ -0,0 +1,164 @@
+///if the pHmeter gives a detailed output
+#define DETAILED_CHEM_OUTPUT 1 
+///if the pH meter gives a shorter output
+#define SHORTENED_CHEM_OUTPUT 0 
+
+/*
+* a pH booklet that contains pH paper pages that will change color depending on the pH of the reagents datum it's attacked onto
+*/
+/obj/item/pHbooklet
```

Comment:
Type names are *required* to be snake_case. While not required for variable names, I'd rather this be changed *everywhere* to just be `ph`. This one specifically would become `ph_booklet`, for example. Think of names like `ai_controller`, where AI is technically an acronym, but is treated as a word.

Reply:
I'll see if I can track down my use of pH, I assumed pH would be easier to search for since there's a lot of ph already in the codebase

Reply:
`ph_` should be an easy search.

Reply:
I think I got them all